aspnetcore/blazor/host-and-deploy/server.md

```diff
@@ -62,91 +62,103 @@ Recommendations for global deployments to geographical data centers:
 
 * Deploy the app to the regions where most of the users reside.
 * Take into consideration the increased latency for traffic across continents. To control the appearance of the reconnection UI, see <xref:blazor/fundamentals/signalr#control-when-the-reconnection-ui-appears>.
-* For Azure hosting, use the [Azure SignalR Service](#azure-signalr-service).
+* For Azure hosting outside of Azure App Service, use the [Azure SignalR Service](#azure-signalr-service).
 
 :::moniker-end
 
-## Azure SignalR Service
+## Azure App Service
 
-:::moniker range=">= aspnetcore-8.0"
+Hosting on Azure App Service requires configuration for WebSockets and session affinity, also called Application Request Routing (ARR) affinity.
 
-For Blazor Web Apps that adopt interactive server-side rendering, consider using the [Azure SignalR Service](xref:signalr/scale#azure-signalr-service). The service works in conjunction with the app's Blazor Hub for scaling up to a large number of concurrent SignalR connections. In addition, the service's global reach and high-performance data centers significantly aid in reducing latency due to geography. If your hosting environment already handles these concerns, using the Azure SignalR Service isn't necessary.
+> [!NOTE]
+> A Blazor app on Azure App Service doesn't require [Azure SignalR Service](#azure-signalr-service).
 
-<!-- UPDATE 9.0 Remove when support is present -->
+Enable the following for the app's registration in Azure App Service:
 
-> [!NOTE]
-> [Stateful reconnect](xref:signalr/configuration#configure-stateful-reconnect) (<xref:Microsoft.AspNetCore.SignalR.Client.HubConnectionBuilderHttpExtensions.WithStatefulReconnect%2A>) was released with .NET 8 but isn't currently supported for the Azure SignalR Service. For more information, see [Stateful Reconnect Support? (`Azure/azure-signalr` #1878)](https://github.com/Azure/azure-signalr/issues/1878).
+* [WebSockets](xref:fundamentals/websockets) to allow the WebSockets transport to function. The default setting is **Off**.
+* Session affinity to route requests from a user back to the same App Service instance. The default setting is **On**.
 
-:::moniker-end
+1. In the Azure portal, navigate to the web app in **App Services**.
+1. Open **Settings** > **Configuration**.
+1. Set **Web sockets** to **On**.
+1. Verify that **Session affinity** is set to **On**.
 
-:::moniker range="< aspnetcore-8.0"
+If you wish to disable the session affinity feature for a Blazor app that runs on a single Azure instance, see [Disable Session affinity cookie (ARR cookie) for Azure web apps](https://azure.github.io/AppService/2016/05/16/Disable-Session-affinity-cookie-(ARR-cookie)-for-Azure-web-apps.html).
```

Comment:
I'm not aware of any benefit of doing this, and the app will break if they later decide to scale it out. I think we can just remove this text.

```suggestion
If you wish to disable the session affinity feature for a Blazor app that runs on a single Azure instance, see [Disable Session affinity cookie (ARR cookie) for Azure web apps](https://azure.github.io/AppService/2016/05/16/Disable-Session-affinity-cookie-(ARR-cookie)-for-Azure-web-apps.html).
```